README.md

```diff
@@ -1 +1,16 @@
+# Postgres backup and restore docker images
+
+## Image origin
+
 FORKED FROM: https://github.com/avencera/postgres-backup-restore that was FORKED FROM: https://github.com/schickling/dockerfiles/
+
+## Usage in the PDS
+
+Remember that this is a public repo.
+We are using only `backup` image in our infrastructure. The `restore` image is supposed to be
+used only in extraordinary situation.
```

Comment:
I wouldn't use "extraordinary", we actually have a "restore" runbook which we can link here and be specific about this.